_posts/2018-07-09-conference-talks-mobile-services.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: news
+section: news
+title: Where you can find talks on AeroGear Mobile Services!
+author: aerogear
+tag: community
+---
+
+Following on from the announcment about the first release of AeroGear Mobile Services on OpenShift we'll be hitting up a few conferences to talk about it some more.
+
+You can find us at:
+
+TODO: add any conference you are confirmed attending to talk about Mobile Services here.
+
+
+
+We're also encouring the AeroGear Team to submit talks to conferences that we are not yet confirmed for. See below the list of conferences that we'd like to also hit.
```

Comment:
Not sure what we mean here - especially with regard to getting approval for travel before submitting to conferences...

We should also aim to come up with some proposed conferences to target, but this may not be something we put up on the site until they are confirmed.

Reply:
ok will remove this section.

Reply:
@johnfriz just revisiting this one. I included this section based on the description/comments of this trello card: https://trello.com/c/sSrVDGaO/31-community-engagement. Not sure if I've misinterpreted your comment there? Do you have a suggestion for a better way to word this?